packages/runner-ct/src/index.js

```diff
@@ -1,2 +1,12 @@
 import '@packages/runner/src/main.scss'
-import './main.tsx'
+
+// in CI we want to continue to use the full (legacy) runner-ct instead of main-lite
+// main-lite does not import everything, but until we get run mode working with the
+// new unification code, we need CI to use the legacy runner-ct
+// new DefinePlugin({ WEBPACK_MAIN_ENTRY: process.env.CI ? './main' : './main-lite' }),
+//
+/// #if process.env.CI
+import './main'
+/// #else
+import './main-lite'
```

Comment:
We build a different, more slim bundle to inject into unified runner that simply bundles the reporter and driver (`main-lite`). `main` is for the legacy CT runner, which we still use in CI for running CT tests (for now).